stockpile/roles/postgresql/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+  # Collect PostgreSQL related configuration
+  # Requires the postgresql.conf to be present
+  # under the /var/lib/pgsql/data directory
+  # TODO: Make this more general to read from
+  #       any location where the configuration
+  #       lives.
+  - name: Get max_connections
+    shell:
+       grep "max_connections = " | cut -d ' ' -f 3
```

Comment:
This assumes you are in the correct directory, right?

Reply:
Thanks for pointing this out, it is a typo. Need to fix